### PR TITLE
Admin Login

### PR DESCRIPTION
**Pull request by KANNAN84-hash**
Here I have done Admin Login feature for Admin to login using email and password

### CODE REVIEW

File: sql/movies.sql [hunk 1]
```diff
@@ -3,4 +3,17 @@ create table movies(movie_id  serial primary key ,movie_name varchar(255) not nu
...
 	insert into movies(movie_name,release_date,end_date) values ('Master','2020-12-01','2021-04-01');
     insert into movies(movie_name,release_date) values ('Karnan','2021-02-01');
-	insert into movies(movie_name,release_date) values ('Eswaran','2020-12-01');
\ No newline at end of file
+	insert into movies(movie_name,release_date) values ('Eswaran','2020-12-01');
+	
+	
+create table users(id serial primary key,name varchar(255) not null ,
+				 email varchar(255) not null unique, password varchar(255) not null,
+				  role varchar(255) not null check(role in('Admin','User')));
```
Comment (csys-fresher-batch-2021) [inline sql/movies.sql]: Prefer - ADMIN, USER

File: src/main/java/in/kannan/dao/AdminDAO.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import in.kannan.exception.AdminLoginException;
+import in.kannan.exception.DBException;
+import in.kannan.model.Users;
+import in.kannan.util.ConnectionUtil;
+
+public class AdminDAO {
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/kannan/dao/AdminDAO.java]: UserDAO

File: src/main/java/in/kannan/dao/AdminDAO.java [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+	/**
+	 * finds the correct admin to login
+	 * 
+	 * @param email
+	 * @param password
+	 * @throws DBException
+	 * @throws AdminLoginException
+	 */
+	public static Users findByGmailAndPassword(String email, String password) throws DBException, AdminLoginException {
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/kannan/dao/AdminDAO.java]: findByEmailAndPassword

File: src/main/java/in/kannan/model/Users.java [hunk 4]
```diff
@@ -0,0 +1,91 @@
+package in.kannan.model;
+
+public class Users {
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/kannan/model/Users.java]: User user  = new User(1,'Kannan'). Rename Users to user.

File: src/main/java/in/kannan/service/AdminService.java [hunk 5]
```diff
@@ -0,0 +1,37 @@
+package in.kannan.service;
+
+import in.kannan.dao.AdminDAO;
+import in.kannan.exception.AdminLoginException;
+import in.kannan.exception.DBException;
+import in.kannan.exception.ServiceException;
+import in.kannan.model.Users;
+import in.kannan.validator.AdminValidator;
+
+public class AdminService {
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/kannan/service/AdminService.java]: UserService

**Review by ghost — CHANGES_REQUESTED** [review]
Changes Requested.

**Review by ghost — APPROVED** [review]

**Comment by KANNAN84-hash** [comment]
Here I have completed the Admin login feature .

**Pull request merged**